docs/src/pages/component-demos/expansion-panels/SimpleExpansionPanel.js

```diff
@@ -0,0 +1,42 @@
+// @flow weak
+
+import React from 'react';
+import PropTypes from 'prop-types';
+import { withStyles, createStyleSheet } from 'material-ui/styles';
+import ExpansionPanel from 'material-ui/ExpansionPanel';
+import Typography from 'material-ui/Typography';
+import ExpandMoreIcon from 'material-ui-icons/ExpandMore';
+
+const styleSheet = createStyleSheet({
+  root: {
+    marginTop: 30,
+    width: '100%',
+  },
+  content: {
+    padding: 24,
```

Comment:
I think that we should be using `theme.spacing.unit` multiple as much as we can. `padding` and `marginTop`.